ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.17__periodics-mce.yaml

```
@@ -46,166 +61,27 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-aws-ovn-conformance
-  cron: 0 4 * * *
+- as: e2e-agent-connected-hub415-mce27-hc417-conformance
```

```suggestion
- as: e2e-powervs-agent-connected-hub415-mce27-hc417-conformance
```
Seems like an atypical config - typically grouped on cloud provider.